rust/gitxetcore/src/data_processing_v2.rs

```diff
@@ -131,7 +131,7 @@ impl PointerFileTranslatorV2 {
             summarydb,
             cas: cas_client,
             prefix: config.cas.prefix.clone(),
-            small_file_threshold: config.cas.size_threshold,
+            small_file_threshold: SMALL_FILE_THRESHOLD,
```

Comment:
We may want to use `config.cas.size_threshold` per the changes in https://github.com/xetdata/xet-core/pull/227